addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.scss

```diff
@@ -29,83 +19,11 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ThreadViewTopbar {
-    background-color: gray('100');
-}
-
-.o_ThreadViewTopbar_button {
-    background: none;
-    border: none;
-    outline: none;
-
-    &.o-active {
-        color: gray('700');
-    }
-
-    &:not(.o-active) {
-        color: gray('500');
-    }
-
-    @include hover-focus () {
-        outline: none;
-        color: gray('600');
-    }
-}
-
 // use selector specificity to override bootstrap style
 input.o_ThreadViewTopbar_guestNameInput {
     width: 150px;
 }
 
-.o_ThreadViewTopbar_invitePopoverView .popover-body {
```

Comment:
Because the PopoverView no longer has the `popover-body` class, I think this rule can be safely removed.

Reply:
Indeed, this was necessary when we used `web.Popover` but now it's `mail.PopoverView` so this is dead code.